prime-dotnet-webapi/Services/Razor/RazorConverterService.cs

```diff
@@ -66,10 +66,9 @@ public async Task<string> RenderTemplateToStringAsync<TModel>(RazorTemplate<TMod
             return output.ToString();
         }
 
-        public async Task<string> RenderEmailTemplateToString<TModel>(EmailTemplateType type, TModel viewModel)
+        public async Task<string> RenderEmailTemplateToString<TModel>(EmailTemplate template, TModel viewModel)
```

Comment:
There is a compile warning ... maybe should simply return `string` now?

Reply:
Good catch. It is fixed.